### ISSUE TITLE
[BUG] A11y - begin test button not accessible, missing labels

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#2268]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

"Begin test" button may not be accessible, its label should be associated with the person's name and birth date

### Steps To Reproduce

Seach on Conduct Tests page, inspect code

### Expected Behavior

Button is labeled for screen readers w/ an aria

### Does this issue have a priority label?

- [X] I have added a priority label

### Has issue been assigned to a team?

- [X] I have added a team label
